setup/install-openroad-rhel7.sh

```diff
@@ -106,11 +108,8 @@ sudo ln -s /usr/bin/tclsh8.6 /usr/bin/tclsh
 sudo ldconfig
 cd -
 
-
-cd ..
-
 # Install OpenROAD tools.
-cd ..
+cd ${scr_path}/..
```

Comment:
```suggestion
cd ../..
```
We should still combine the two `cd ..` steps to match the earlier `cd tcl/unix` change, though. Thank you for cleaning that up.